packages/api/blocks-api/package.json

```diff
@@ -28,7 +28,7 @@
         "test:watch": "jest -w"
     },
     "peerDependencies": {
-        "@nestjs/common": "^8.0.0",
```

Comment:
Could we support v8 and v9? Then our developers wouldn't need to directly update Nest when updating to the newest library version

Reply:
No I dont think this is possible, then we have to support two graphql modules one with driver (v10) and one without (v9)